src/pages/home/report/ReportActionItemMessageEdit.tsx

```diff
@@ -406,6 +406,11 @@ function ReportActionItemMessageEdit(
                             style={[styles.textInputCompose, styles.flex1, styles.bgTransparent]}
                             onFocus={() => {
                                 setIsFocused(true);
+                                InteractionManager.runAfterInteractions(() => {
+                                    requestAnimationFrame(() => {
+                                        reportScrollManager.scrollToIndex(index, true);
+                                    });
+                                });
                                 reportScrollManager.scrollToIndex(index, true);
```

Comment:
Is there a need to call `scrollToIndex` here again even after calling within `requestAnimationFrame`?